Projects/App/Sources/UI/Organization/OrgDetail/View/OrgDatailInformationView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  OrgDatailCell.swift
+//  App
+//
+//  Created by Keum MinSeok on 2022/10/25.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import DesignSystem
+import SnapKit
+
+final class OrgDetailInformationView: UIView {
+
+    // MARK: - Properties
+    
+    let informationLabel = UILabel()
+    let numberLabel = UILabel()
+
+    // MARK: - LifeCycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureUI()
+        createLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    // MARK: - Function
+    
+}
+
+// MARK: - UI Function
+
+extension OrgDetailInformationView {
+
+    private func configureUI() {
+        backgroundColor = .zestyColor(.background)
+        
+        informationLabel.textColor = .zestyColor(.gray3C3C43)
```

Comment:
p1;

요 친구는 .secondaryLabel이라고 UIColor에 기본으로 있는 색상을 사용합니다!
```suggestion
        informationLabel.textColor = .secondaryLabel
```

Reply:
[a313180](https://github.com/DeveloperAcademy-POSTECH/MacC-TEAM-ZESTY/pull/70/commits/a313180321e8d689519402a60e6cc20ef3304e13) 색상 변경 완료하였습니다 ~